src/globus_sdk/paging/table.py

```diff
@@ -9,13 +9,13 @@
 # stub for mypy
 class _PaginatedFunc:
     _has_paginator: bool
-    _paginator_class: Type[Paginator]
+    _paginator_class: Type[Paginator[Any]]
```

Comment:
Could these be bound to some `Type[Paginator[PaginatedT]]` where `PaginatedT` is a `PageT` iterator or something like that?

Reply:
I've put the test case we discovered for why `Paginator[Any]` is bad into the commit message for the fix.